spec.html

```diff
@@ -820,6 +820,74 @@ <h1>
           1. Return _asyncContextMapping_.
         </emu-alg>
       </emu-clause>
+
+      <emu-clause id="sec-asynccontext-wrappedfunctioncreate" type="abstract operation">
+        <h1>
+          AsyncContextWrappedFunctionCreate (
+            _targetFunction_: a function object,
+            _snapshot_: a List of Async Context Mapping Records,
+          ): either a normal completion containing a function object or a throw completion
+        </h1>
+        <dl class="header">
+          <dt>description</dt>
+          <dd>It is used to specify the creation of new AsyncContext wrapped function exotic objects.</dd>
+        </dl>
+        <emu-alg>
+          1. Let _proto_ be ? <emu-meta effects="user-code">_targetFunction_.[[GetPrototypeOf]]</emu-meta>().
+          1. Let _internalSlotsList_ be the list-concatenation of « [[Prototype]], [[Extensible]] » and the internal slots listed in <emu-xref href="#table-internal-slots-of-asynccontext-wrapped-function-exotic-objects"></emu-xref>.
```

Comment:
This table of internal slots is missing.

Reply:
Fixed.